Kickstarter-iOS/Views/Controllers/MessageBannerViewController.swift

```diff
@@ -30,6 +30,16 @@ final class MessageBannerViewController: UIViewController, NibLoading {
     static let showDuration: TimeInterval = 0.3
   }
 
+  override func viewDidLoad() {
+    super.viewDidLoad()
+
+    _ = self.backgroundView
```

Comment:
Moving this to `viewDidLoad` to better separate concerns (making components accessible vs setting styles)